releasenotes/notes/change-metadata-interval-0dc52511f61ebfd1.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
```

Comment:
nit: I don't think this really qualifies as a `feature`, I'd put this under `enhancement` :)